docs/Guidelines/Brand/brand.stories.mdx

```diff
@@ -0,0 +1,125 @@
+import { Meta, Story, Canvas } from '@storybook/addon-docs'
+import { Image } from '../../../src/components/Image'
+import { color } from '../../../src/theme'
+
+<Meta
+  parameters={{
+    previewTabs: {
+      'storybook/docs/panel': {
+        hidden: true,
+      },
+    },
+  }}
+  title="Guidelines/Brand"
+  decorators={[]}
+/>
+
+# Brand
+
+<Image
+  style={{
+    backgroundColor: 'rgba(0, 182, 240, 0.5)',
+    width: '100%',
+    height: '275px',
+    marginBottom: '2rem',
+    borderRadius: '4px',
+  }}
+  src="https://www.ticketswap.com/static/images/logo/logo-blueprint.svg"
+/>
+
+## Manifesto
+
+The passion and enthusiasm the founders and
+the employees of TicketSwap have for music,
+dance, culture, arts, sports and any talent-filled
+entertainment is what drives our machine.
+That’s why we are fans for the fans. We want to
+cultivate the spices of life we get from watching
+our favorite singer on stage, our favorite DJ in an
+underground club, and favorite ballerina taking
+flight. When tickets to events sell-out, we are
+there to help you find second-hand tickets.
+When you can’t see your favorite concert
+because something unexpectedly turns up,
+we help you find another fan in common who
+will buy your ticket from you and enjoy the show
+as much as you would have.
+We combine our passion for fans and music
+events with our own technology to create a fair,
+honest and safe place for people to resell their
+tickets online. By placing a selling-limit of no
+more than 20% above the original ticket price,
+we aim to stop profiteering, fraud, keep ticket
+reselling fair, and help those fans who want to go
+to an event without getting ripped off.
+
+We started TicketSwap with simplicity in mind.
+Our platform will remain as simple as possible
+for anyone to enter the site, buy sell and place
+wanted listings for second-hand tickets.
+
+Our business values will remain being customer-
+driven, honest, safe, transparent, and simple.
+
+The journey we are on is not only to pursue
+technological advances and business growth:
+we want to change the world. By this we mean
+we will stick to our values in order to change
+the way the world sees the secondary ticketing
+industry: we will bring fairness to the market,
+and we will continue to lobby policy-makers,
+sit with event organizers, engage with artists
+and preach the gospel of ticketing to the fans.
+After all, we are fans too.
+Sound ambitious? Always.
+We will never rest until the world knows
+TicketSwap is here for you, the fans.
+
+## Vision
+
+We’re in business to make the lives of fans better.
+
+## Design principles
```

Comment:
It's empty? 😢

Reply:
> Why are some colors excluded from this part but included in the one below it?

@Roohn idk, like I said. I used the Brandbook as my reference. What colors are you referring to? The socials? Technically those aren't our colors right?

Reply:
I mean we are missing `sky` and `nova` (and socials but those are indeed not our colors)

Reply:
@Roohn Do you know if these colors were added in a later stage? Trying to figure out if they were left out on purpose or that this document is just outdated.

Reply:
Nova was a recent addition. Sky I'm not sure about 🤔

Reply:
@Roohn design is also going to discuss this with marketing? I'll leave it like this for now.